Server/Components/Pawn/Scripting/Player/Natives.cpp

```diff
@@ -253,8 +253,21 @@ SCRIPT_API(ResetPlayerMoney, bool(IPlayer& player))
 
 SCRIPT_API(SetPlayerName, int(IPlayer& player, const std::string& name))
```

Comment:
Maybe add SCRIPT_API_RETVAL(-1 here to return -1 if player is invalid

Reply:
Do you mean SCRIPT_API_FAILRET?

In SA:MP it returns 0 if player is invalid.

Reply:
OK no change needed then